pkg/executables/kubectl.go

```diff
@@ -2535,3 +2536,20 @@ func (k *Kubectl) DeleteCRD(ctx context.Context, crd, kubeconfig string) error {
 
 	return nil
 }
+
+// GetEKSARelease returns an eksareleases CR on the cluster.
+func (k *Kubectl) GetEKSARelease(ctx context.Context, releaseName string, kubeconfigFile string) (*releasev1alpha1.EKSARelease, error) {
```

Comment:
what do you think about instead of adding this method here, injecting a `kubernetes.Client` in the validations code, so you can just use `Get`?

Reply:
I don't have any strong opinions about it. Is there a particular reason we don't want to add this method? 
For the time being, I can remove it and replace it with GetObject instead.

Reply:
This pattern where we add accessor methods (get, list) and writer methods (create, update) for every single resource doesn't scale. Just look at the sheer size of `kubectl.go`. Not only requires custom code on the implementation side but it also requires updating the receiving interfaces (and their mocks) every time we add/change one of these methods. Moreover, testing code that uses this type of client can be quite cumbersome, needing custom mocks and specific EXPECT calls for every single read or write. Instead, testing using a FakeClient is way easier.

AFAIK GetObject doesn't behave the same way as the client, since it requires to pass the resource type.